src/incubator/TextField/Input.tsx

```diff
@@ -69,6 +68,10 @@ const styles = StyleSheet.create({
         textAlignVertical: 'center'
       }
     })
+  },
+  webStyle: {
```

Comment:
Does this solve your issue? I believe we meant to get the 'webStyle' to a const outside of the stylesheet, as the stylesheet is typed. You can move it outside the component or create a memorized const as this is not a dynamic style

Reply:
Yes it solves the issue (with the ts-expect-error) and keeps the style in the stylesheet so I think it is more readable and clean.